ibridgesgui/__main__.py

```diff
@@ -302,5 +302,10 @@ def main():
     app.exec()
 
 
-if __name__ == "__main__":
+def main_deprecated():
+    """Deprecate the ibridges-gui call."""
+    print("'ibridges-gui' will be deprecated. Please use 'ibridges gui' from ibridges 1.5.1")
```

Comment:
You can use a DeprecationWarning, then it will also show up in pytest and the like!

Reply:
Also, personally, I would give people a little bit more time and keep the deprecated command until ibridges 2.0.

Reply:
Oh, that is a misunderstanding. I meant the new way of calling the gui is available with ibridges 1.5.1 or higher. I would also like to give people several months to adopt it.

Reply:
I added it to our normal iBridges GUI log file. Since the tab "log" just shows the log entries as soon as you connect to iRODS the message will not be shown there. It is in the log and it will be printed on the shell.